doc/source/cluster/kubernetes/getting-started/raycluster-quick-start.ipynb

```diff
@@ -0,0 +1,758 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "9ef5b3ae-3404-4bb5-8894-f2719f39f4d6",
+   "metadata": {
+    "editable": true,
+    "slideshow": {
+     "slide_type": ""
+    },
+    "tags": []
+   },
+   "source": [
+    "(kuberay-operator-deploy)=\n",
+    "## Step 2: Deploy a KubeRay operator\n",
+    "\n",
+    "Deploy the KubeRay operator with the [Helm chart repository](https://github.com/ray-project/kuberay-helm) or Kustomize.\n",
+    "\n",
+    "### Method 1: Helm"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "648041da-62ac-45b3-bf9e-8bd6433f94d7",
+   "metadata": {
+    "editable": true,
+    "slideshow": {
+     "slide_type": ""
+    },
+    "tags": [
+     "nbval-ignore-output",
+     "remove-output"
+    ]
+   },
+   "outputs": [],
+   "source": [
+    "helm repo add kuberay https://ray-project.github.io/kuberay-helm/\n",
+    "helm repo update"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 42,
+   "id": "d48de329-1b64-4aa6-9ec5-33df23b4e2bb",
+   "metadata": {
+    "editable": true,
+    "slideshow": {
+     "slide_type": ""
+    },
+    "tags": [
+     "nbval-ignore-output",
+     "remove-output"
+    ]
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "NAME: kuberay-operator\n",
+      "LAST DEPLOYED: Tue Mar 11 21:50:15 2025\n",
+      "NAMESPACE: default\n",
+      "STATUS: deployed\n",
+      "REVISION: 1\n",
+      "TEST SUITE: None\n"
+     ]
+    }
+   ],
+   "source": [
+    "# Install both CRDs and KubeRay operator v1.3.0.\n",
+    "helm install kuberay-operator kuberay/kuberay-operator --version 1.3.0"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "4a4b8777-9fd4-4eea-a97d-0a40e4762864",
+   "metadata": {
+    "editable": true,
+    "slideshow": {
+     "slide_type": ""
+    },
+    "tags": []
+   },
+   "source": [
+    "### Method 2: Kustomize"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "03d55b9c-0479-4879-b9bd-e707d5a91391",
+   "metadata": {
+    "editable": true,
+    "slideshow": {
+     "slide_type": ""
+    },
+    "tags": [
+     "nbval-skip",
+     "remove-output"
+    ]
+   },
+   "outputs": [],
+   "source": [
+    "# Install CRD and KubeRay operator.\n",
+    "kubectl create -k \"github.com/ray-project/kuberay/ray-operator/config/default?ref=v1.3.0\""
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "d4442614-b6c1-46ae-8c39-5b49c600806f",
+   "metadata": {
+    "editable": true,
+    "slideshow": {
+     "slide_type": ""
+    },
+    "tags": []
+   },
+   "source": [
+    "### Validate Installation\n",
+    "\n",
+    "Confirm that the operator is running."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 43,
+   "id": "a94535d4-195a-4662-9e98-9304cb93e50f",
+   "metadata": {
+    "editable": true,
+    "slideshow": {
+     "slide_type": ""
+    },
+    "tags": [
+     "nbval-ignore-output",
```

Comment:
Why do we ignore the output here?

Reply:
The original doc also does not include the `kubectl wait` command. However, this command is needed for the doc test, so I ignore the output.